### DISCUSSION TITLE
What's the difference between tables and records?

### DISCUSSION THREAD

**Discussion by vwkd** [nushell/nushell#11759] [Q&A]
The [book](https://www.nushell.sh/book) starts off with a nice explanation on tables and how to use them. However, the following pages then throw you into cold water by silently using records. For example, in [Quick Tour](https://www.nushell.sh/book/quick_tour.html) the command `sys | get host` returns a record.

What's a table? What's a record? What's the difference between tables and records? When and how are they used?

It would be great if the book did a better job introducing the different data types, and what you need to know about them.

**Comment by fdncred** [comment]
A table is a list of records.
